xml/art_sle_ha_pmremote.xml

```diff
@@ -274,50 +301,213 @@
+      connectivity:
+     </para>
+     <screen>&prompt.root;<command>ssh</command> -p 3121 &node4;</screen>
+     <para>From the output you can tell if your connection works:</para>
+     <variablelist>
+      <varlistentry>
+       <term>Working connection</term>
+       <listitem>
+        <screen>ssh_exhange_identification: read: Connection reset by peer.</screen>
+       </listitem>
+      </varlistentry>
+      <varlistentry>
+       <term>Broken connection</term>
+       <listitem>
+        <screen>ssh: connect to host &node4; port 3121: No route to host
+ssh: connect to host &node4; port 3121: Connection refused</screen>
+        <para>If you see either of the previous messages, the connection does
+        not work. Use the <option>-v</option> option for <command>ssh</command>
+         and execute the command again to see debugging messages. This could
+         be helpful to find connection, authentication, or configuration
+         problems. Multiple <option>-v</option> options increase the verbosity.
+        </para>
+       </listitem>
+      </varlistentry>
+     </variablelist>
+     <para>If needed, add more guest nodes and configure them as described above.</para>
+    </step>
+    <step>
+     <para>
+      Shut down the KVM guest and proceed with <xref
+       linkend="sec.ha.pmremote.config.uc1.integrate"/>.
+     </para>
+    </step>
+   </procedure>
+  </sect2>
+  <sect2 xml:id="sec.ha.pmremote.config.uc1.integrate">
+   <title>Integrating KVM Guests into Cluster</title>
+   <para>To integrate the KVM guests into the cluster, perform the following
+    steps on the physical host <systemitem class="domainname">&node1;</systemitem>:</para>
+   <procedure>
+    <step>
+     <para>Start &pace;:</para>
+     <screen>&prompt.root;<command>systemctl</command> start pacemaker</screen>
+    </step>
+    <step>
+     <para>Check if your KVM guest node and IP address is listed
+     in the file <filename>/etc/hosts</filename>.</para>
+    </step>
+    <step>
+     <para>
+      Create a <systemitem>ocf:pacemaker:remote</systemitem> primitive:
+     </para>
+     <remark>toms 2017-07-11: Check the interval values</remark>
+     <screen>&prompt.root;<command>crm</command> configure
+&prompt.crm.conf;<command>primitive</command> &node3; ocf:pacemaker:remote \
+     params server=&node3; recommection_interval=10 \
+     op monitor interval=
+&prompt.crm.conf;<command>commit</command></screen>
+    </step>
+    <step>
+     <para>Check the status of the cluster with the command
+      <command>crm status</command>. It
+      should contain a running cluster with nodes that are all accessible.
+     </para>
+    </step>
+   </procedure>
+  </sect2>
+ </sect1>
+
+ <sect1 xml:id="sec.ha.pmremote.install.virt-guest-nodes">
+  <title>Setting Up a Cluster with Virtual Guest Nodes</title>
+  <remark>toms 2017-07-11: Are other virtualization techniques supported, like
+  XEN...?</remark>
```

Comment:
Yes. As we know libvirt also supports Xen, which means we can use the RA ocf:heartbeat:VirtualDomain to manage Xen guests as well. But of course we can either use the RA ocf:heartbeat:Xen to do that.